index.js

```diff
@@ -477,6 +477,9 @@ module.exports = function (config, deps) {
         .end(
         function (err, res) {
           if (err != null) {
+            if (err.status !== 201) {
```

Comment:
@krystophv Do you think we can not put in this if-statement, and just return `err` instead of building a new Error object? There is valuable tracing info from platform in `err` that is thrown away if we do this.